Use clang tidy to update codes for readability.

imikejackson opened the pull request:
clang-tidy was run with the following arguments:
-checks='-*,+readability-*,-readability-identifier-naming,modernize-use-equals-default'

Signed-off-by: Michael Jackson <mike.jackson@bluequartz.net>

mmarineBlueQuartz commented on Source/SIMPLib/Common/PhaseType.h:
```diff
@@ -83,7 +83,7 @@ class SIMPLib_EXPORT PhaseType
    * @return
    */
-  static Types FromQVector(QVector<EnumType> &types);
+  static Types FromQVector(QVector<EnumType>& vec);
```
types has more meaning than vec.  It's minor, but this probably shouldn't be changed.

mmarineBlueQuartz commented on Source/SIMPLib/Common/ShapeType.h:
```diff
@@ -87,7 +87,7 @@ class SIMPLib_EXPORT ShapeType
    * @return
    */
-  static Types FromQVector(QVector<EnumType> &types);
+  static Types FromQVector(QVector<EnumType>& vec);
```
types has more meaning than vec.  It's minor, but this probably shouldn't be changed.

mmarineBlueQuartz commented on Source/SIMPLib/Geometry/RectGridGeom.h:
```diff
@@ -56,10 +56,10 @@ class SIMPLib_EXPORT RectGridGeom : public IGeometryGrid
+
+    void setXBounds(FloatArrayType::Pointer xBnds);
+    void setYBounds(FloatArrayType::Pointer yBnds);
+    void setZBounds(FloatArrayType::Pointer zBnds);
```
xBounds is a lot more readable than xBnds.  It's minor, but this probably shouldn't be changed.

imikejackson committed:
Use clang tidy to update codes for readability.

clang-format was also run on all changed codes

clang-tidy was run with the following arguments:
-checks=-*
,modernize-use-equals-default
,readability-avoid-const-params-in-decls
,readability-braces-around-statements
,readability-container-size-empty
,readability-delete-null-pointer
,readability-else-after-return
,readability-implicit-bool-conversion
,readability-redundant-control-flow
,readability-redundant-declaration
,readability-redundant-function-ptr-dereference
,readability-redundant-member-init
,readability-redundant-smartptr-get
,readability-redundant-string-cstr
,readability-redundant-string-init
,readability-simplify-boolean-expr
,readability-simplify-subscript-expr
,readability-static-accessed-through-instance
,readability-static-definition-in-anonymous-namespace
,readability-string-compare

Signed-off-by: Michael Jackson <mike.jackson@bluequartz.net>
